enacts/onset/layout.py

```diff
@@ -277,12 +280,26 @@ def controls_layout(lat_min, lat_max, lon_min, lon_max, lat_label, lon_label):
                         ),
                         html.P(
                             Sentence(
-                                Number("probExcThresh1", 30, min=0),
-                                html.Span(id="pet_units"),
+                                Number("prob_exc_thresh1", 30,  min=0),
+                                html.Span(id="pet_units1"),
```

Comment:
I tried that at first. I tried to put the Sentence in a div, span or p to call it back. But I failed. I think it may have to do with that fact that Number has an id (which we need) and is inside Sentence that would have an id to be called back to be generating, among other things, Number's id. And that is not allowed, right? Or I forgot how to deal with that.

Reply:
Not sure I understand the problem. In the layout, put a `P` with an id but no children. Have a callback that sets that `P`'s children to a `Sentence` containing a `Number` and a `Span`. I don't see why the `Number` and `Span` need ids, as long as they share a common ancestor that has an id.

Reply:
I see. I can do that however controls.py's Number's id is not optional and so I would have to set it up so. Just double checking that it's going to be ok (as I am not the original author of controls.py)

Reply:
This links with issue #270

Reply:
Now I see a problem with my suggestion. One or more callbacks is going to declare the Number as an input; if the Number doesn't  exist in the initial layout (because it's generated by a callback), then there will be an error. That error message can be suppressed, but I don't want to do that unless it's necessary, because it sometimes points out real problems.

I think we should have this in the layout
```
Sentence(
    Number(id='prob_exc_thresh'),
    Span(id='prob_exc_units'),
)
```
and let the Number's default value and the Span's children be set by a callback. Currently that's not possible because `default` is mandatory for Number, but we can change that. Let the Number input have no value until the callback is called.

Reply:
Being refreshed about the matter, there is actually another potential argument in having the 3 Setences-with-Number. That would be because the 3 variables for which probability of exceeding can be computing don't have the same domains of values. So it would not be too great not to update the pet value when selecting from one to another. That would mean resetting to some default value each time one returns to selecting that option. And so that means destroying potentially previous choices made for that option (which is not nice).
I don't know if this is very intelligible. But basically I am saying the having 3 of them allows to "save" those choices should a user return to that option.

Reply:
OK, that makes sense.

I would prefer meaningful ids like `pet_input_wrapper_search_days` instead of 1, 2, 3.

Reply:
I hadn't thought about that. The names in the URL don't _have_ to be the same as the component ids, but having them be the same does make things easier to keep track of.  Shorter is better if you can find shorter names that are still meaningful.